app/controllers/sulten/lyche_controller.rb

```diff
@@ -59,13 +65,23 @@ def reservation
         end
       end
 
+      # Check if food is open, if food type is selected
+      @in_food_closed_period = false
+      @reservation_type = request[:reservation_type_id]
+      reservation_type_name = Sulten::ReservationType.find_by(id: @reservation_type).name
+      if reservation_type_name.include?('Mat') && @food_closed
+        if stop < food_open_date
+          @in_food_closed_period = true
```

Comment:
Burde dette være start ikke stop?

Reply:
Det er veldig fucked, men start og stop er to identiske variabler.
Burde kanskje bytte den ut for at navnet skal gi mer mening.

Utenom det, så ser det ut som at de har glemt å definere tidsrom idet de opprettet den stop variabelen... hehehe